packages/story-editor/src/components/library/panes/media/common/mediaElement.js

```diff
@@ -227,9 +225,7 @@ function Element({
           active={active}
         />
         {attribution}
-        {(!src ||
-          isBlobURL(src) ||
```

Comment:
The is blob url check is a good way to see if it something is local. Why remove it?

Reply:
If we want to know whether a resource is not yet uploaded we have utilities like `isResourceProcessing` & co.

Reply:
In my testing, the progress bar took half a second to show, as the media library and queue were not in sync. This check means that the progress bar show, straight away, instead of half a second later. 

It is an extremely inexpensive check, why not leave it in.